### PR TITLE
[WIP] ChromeDriver should be downloaded automatically 

### PR DESCRIPTION
**Pull request by ghostbuster91**
It shouldn't require any manual setup from developers

### CODE REVIEW

**Comment by adamw** [comment]
Thanks!

File: build.sbt [hunk 1]
```diff
@@ -110,7 +103,37 @@ val browserTestSettings = Seq(
       },
       org.scalajs.jsenv.selenium.SeleniumJSEnv.Config().withKeepAlive(debugging)
     )
-  }
+  },
+  downloadLatestChromeDriver := {
+    if (java.nio.file.Files.notExists(new File("target", "chromedriver").toPath)) {
```
Comment (LGLO) [inline build.sbt:108]: I'm afraid this won't work on Windows, where filename is `chromedriver.exe`

File: build.sbt [hunk 2]
```diff
@@ -491,7 +514,6 @@ lazy val httpClientBackend = (projectMatrix in file("httpclient-backend"))
     }
   )
   .jvmPlatform(scalaVersions = List(scala2_13, scala3))
-  .settings(onlyJava11)
```
Comment (LGLO) [inline build.sbt:494]: Won't it break build for users of JDK 8? I think they'll have red builds.

Comment (adamw) [reply build.sbt:494]: Yes, we require java11 for builds now. One of the modules required it anyway.

File: build.sbt (hunk 1 above)
Comment (adamw) [reply build.sbt:108]: Good point, do we have anybody with windows to fix this?:)

**Pull request merged**